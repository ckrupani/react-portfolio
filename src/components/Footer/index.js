import React from "react";

function Footer() {
    return (
        <footer>
            <ul className="flex-row flex-center">
                <li className="mx-2">
                    <a
                        href="https://github.com/ckrupani"
                        target="_blank"
                        rel="noreferrer"
                    >
                        Github
                    </a>
                </li>
                <li className="mx-2">
                    <a
                        href="https://au.linkedin.com/in/chirag-rupani-3309a835"
                        target="_blank"
                        rel="noreferrer"
                    >
                        LinkedIn
                    </a>
                </li>
                <li className="mx-2">
                    <a
                        href="https://twitter.com/ckrupani"
                        target="_blank"
                        rel="noreferrer"
                    >
                        Twitter
                    </a>
                </li>
            </ul>
        </footer>
    );
}

export default Footer;
