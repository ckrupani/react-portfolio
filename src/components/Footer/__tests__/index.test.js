import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Footer from "..";

afterEach(cleanup);

describe("Footer component", () => {
    // First Test
    it("renders", () => {
        render(<Footer />);
    });

    // Second Test
    it("matches snapshot DOM node structure", () => {
        const { asFragment } = render(<Footer />);
        expect(asFragment()).toMatchSnapshot();
    });
});
