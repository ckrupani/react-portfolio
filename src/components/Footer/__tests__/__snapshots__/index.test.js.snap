// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <footer>
    <ul
      class="flex-row flex-center"
    >
      <li
        class="mx-2"
      >
        <a
          href="https://github.com/ckrupani"
          rel="noreferrer"
          target="_blank"
        >
          Github
        </a>
      </li>
      <li
        class="mx-2"
      >
        <a
          href="https://au.linkedin.com/in/chirag-rupani-3309a835"
          rel="noreferrer"
          target="_blank"
        >
          LinkedIn
        </a>
      </li>
      <li
        class="mx-2"
      >
        <a
          href="https://twitter.com/ckrupani"
          rel="noreferrer"
          target="_blank"
        >
          Twitter
        </a>
      </li>
    </ul>
  </footer>
</DocumentFragment>
`;
